src/codeu/chat/server/Server.java

```diff
@@ -81,7 +81,30 @@ public Server(final Uuid id, final Secret secret, final Relay relay,final File l
     this.localFile = new LocalFile(new File(file.getPath()));//file path is given by user
     this.controller = new Controller(id, model,localFile);//Use the new constructor to create this new controller.
     this.relay = relay;
-
+    this.commands.put(NetworkCode.CONVERSATION_AUTHORITY_REQUEST, new Command()
```

Comment:
Stick this added chunk at the bottom of the this.commands.put(NetworkCode... section. If you notice, they're all grouped by Network Code number, so putting a big number in front of a small number like this is a little awkward.